modules/grpc/otel/otel-protobuf-parser.cpp

```diff
@@ -1311,6 +1313,7 @@ syslogng::grpc::otel::ProtobufParser::set_syslog_ng_address(LogMessage *msg, GSo
       sin.sin_port = htons(port);
       *sa = g_sockaddr_inet_new2(&sin);
     }
+#if SYSLOG_NG_ENABLE_IPV6
```

Comment:
`syslog-ng.h` may be worth including directly just to make sure the macro will always be defined.

Reply:
you might be right, but should not it already be included via 

`#include "otel-protobuf-parser.hpp"`
|
V
`#include "logmsg/logmsg.h"`

chain?

i just cannot see the case when it will not have effect

Reply:
Yes. Transitive dependencies may change easily. My suggestion would be just basic precaution and expressing dependencies directly.

Reply:
I see your point, but what I currently see is that all the cpp modules rely on such a transitive dependency, none of them including `syslog-ng.h` directly, they either use their own `.hpp` header to add it or, like I did now, includes it via other headers (transitively)
I choose the first one now as it is the most common habit currently project-wide.